lib/entities/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { isNil, isObject, omitBy } from 'lodash'
+
+export type LocalizedEntity<
+  Entity,
+  LocalizedFields extends keyof Entity,
+  Locales extends keyof any
+> = {
+  [K in keyof Entity]: K extends LocalizedFields ? { [Locale in Locales]: Entity[K] } : Entity[K]
+}
+
+export function toUrlParams(params: Record<string, string | number> | undefined = {}) {
+  const urlQuery = new URLSearchParams()
+
+  Object.entries(sanitizeParams(params)).forEach(([key, value]) => {
+    urlQuery.set(key, `${value}`)
+  })
+
+  const result = urlQuery.toString()
+  return result ? `?${result}` : ''
+}
```

Comment:
curious shouldn't we already have examples for query params in this repo?

Reply:
c&p tricked me, we can handle this more elegant.

Reply:
@momenzadehvida what do you think about this?

Reply:
couple questions:
- can you point me to where this util is being used?
- can we use an off the self solutions with the `qs` lib like this instead? https://github.com/contentful/contentful-sdk-core/blob/0afab0fb02a05eaddd136d0679cdcfaaf448c6f5/src/create-http-client.ts#L100

Reply:
This function is not used, I removed. it looks axios is supporting this. 
https://github.com/axios/axios?tab=readme-ov-file#request-config